test/augmentation/test_augmentation.py

```diff
@@ -4187,3 +4189,55 @@ def test_feature_median_blur(self, device, dtype):
         )
 
         utils.assert_close(out, expected)
+
+
+class TestRandomDownScale:
```

Comment:
from kornia.testing import BaseTester
```suggestion
class TestRandomDownScale(BaseTester):
```

Check out more about our test guide on https://github.com/kornia/kornia/blob/master/CONTRIBUTING.md#coding-standards